docs/package.json

```diff
@@ -38,6 +38,7 @@
   },
   "dependencies": {
     "@fluid-tools/api-markdown-documenter": "^0.6.0",
+    "@fluid-tools/markdown-magic": "file:../tools/markdown-magic",
```

Comment:
This would actually be the first use (to my knowledge) of this syntax in our repo, but it is valid, yes. Elsewhere we lean on `lerna` to do the linking, which allows linking to private packages. But since this is a stand-alone package, outside of our lerna-managed release groups, this seemed like a reasonable route to take (at least for now).

Interested in @tylerbutler's thoughts on this though.